src/main/java/com/celements/model/access/DefaultXWikiDocumentCreator.java

```diff
@@ -20,10 +20,10 @@
   private ModelContext context;
 
   @Override
-  public XWikiDocument createWithoutDefaults(DocumentReference docRef) {
+  public XWikiDocument createWithoutDefaults(DocumentReference docRef, String lang) {
```

Comment:
please readd old api createWithoutDefaults(DocumentReference) and mark deprecated.